dali/operators/image/distortion/jpeg_compression_distortion_op_gpu.cu

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <vector>
+#include "dali/operators/image/distortion/jpeg_compression_distortion_op.h"
+#include "dali/kernels/imgproc/jpeg/jpeg_distortion_gpu_kernel.h"
+#include "dali/kernels/kernel_manager.h"
+
+namespace dali {
+
+class JpegCompressionDistortionGPU : public JpegCompressionDistortion<GPUBackend> {
+ public:
+  explicit JpegCompressionDistortionGPU(const OpSpec &spec)
+    : JpegCompressionDistortion(spec) {
+      kmgr_.Initialize<JpegDistortionKernel>();
+      kmgr_.Resize<JpegDistortionKernel>(1, 1);
+  }
+
+  using Operator<GPUBackend>::RunImpl;
+  ~JpegCompressionDistortionGPU() override = default;
+  DISABLE_COPY_MOVE_ASSIGN(JpegCompressionDistortionGPU);
```

Comment:
This is also redundant - if the base class is non-copyable, so will be the derived class.